src/consumer/frame.js

```diff
@@ -141,6 +149,26 @@ class Frame extends EventEmitter {
     if (this.wrapper.parentNode === this.container) {
       this.container.removeChild(this.wrapper);
       this.emit('xfc.unmounted');
+      this.cleanup();
+    }
+  }
+
+  /**
+   * Cleans up references of detached nodes by setting them to null
+   * to avoid potential memory leak
+   */
+  cleanup() {
```

Comment:
Users may create tons of global variables that reference created frames and iframes and wrappers will be dangling in memory even after calling unmount method in old code. Though it's very unlikely to happen, this cleanup method ensures that we won't have memory leak issues even that does happen.